rgo/git.py

```diff
@@ -153,11 +155,11 @@ def describe(self, pkg=None, spec_version=None):
         # is done after a squash merge that reduces number of commits.
         date = datetime.datetime.now().strftime("%Y%m%d%H%M%S")
 
+        LOGGER.info("Using version from {!s}".format(version_from))
         if spec_version is not None and spec_version != git_version:
-            # versions in git and spec and differ
-            # we prefer version from the spec in order to keep rpm dependencies working
-            LOGGER.warning("Versions in git and spec file differ, Using version from the spec file: %r", spec_version)
+            LOGGER.warning("Versions in git ({!s}) and spec file ({!s}) differ".format(git_version, spec_version))
```

Comment:
I'd like to have a single logged line instead of (potentially) two here. I don't think it's that important to warn about the version being different, it's a normal state of things most of the time. IMO, not enough attention is usually paid to log lines being terse and concise :slightly_smiling_face: 

Why not just log a single line like so:

> Using version from git: X.Y.Z

(putting the actual version used in the log line)

Reply:
Done. I moved the logging line to the end of the function to keep the code simple (the 'version' variable is resolved at that time and no additional `if` blocks are needed).